### PR TITLE
New sanitizers and a session API

### PR DESCRIPTION
**Pull request by suutari-ai**
Add sanitation sesssion API and some new sanitizers so it is possible to
use the database sanitizer without implementing this basic stuff
yourself.

### CODE REVIEW

File: database_sanitizer/sanitizers/constant.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+def sanitize_empty_json_dict(value):
+    return '{}'
+
+
+def sanitize_empty_json_list(value):
+    return '[]'
+
+
+def sanitize_invalid_django_password(value):
+    return '!'
```
Comment (JsseL) [inline database_sanitizer/sanitizers/constant.py]: Nice. Didn't know about this. Good to have it.

**Comment by JsseL** [comment]
Nice stuff. This makes the package a bit more "batteries included". :shipit:

**Comment by frwickst** [comment]
@suutari-ai One thing that I though of now. Could you write some documentation regarding the session API? Even if it is just in the source files for now. I'm mainly thinking of the `hash_text_to_ints` function, how it works and how to use it. It might be really clear to you right now, but it was not instantly clear how you used it at first glance.

**Comment by suutari-ai** [comment]
@frwickst You're right. I'll add a docstring to `hash_text_to_ints` etc.

**Comment by suutari-ai** [comment]
Will do that in a separate PR though.

**Pull request merged**